hibike/devices/YogiBear/YogiBear.cpp

```diff
@@ -63,43 +64,19 @@ void loop() {
 
 uint32_t device_write(uint8_t param, uint8_t* data, size_t len) {
   switch (param) {
-    case ENABLE:
```

Comment:
In future, it'd be nice if the pull request summary lists the params changed. Makes it easier on the reviewer to not have to hunt down explicitly which ones.